### ISSUE TITLE
Skip To Content shows up when Code Maroon Alert is visible.

### ISSUE THREAD

**Issue by kaladay** [TAMULib/weaver-components#511]
During a Code Maroon Alert, the Skip To Content link is visible.

This should be fixed in a way such that it still passes Accessibility requirements.
